GDevelop.js/__tests__/GDJSBasicCodeGenerationIntegrationTests.js

```diff
@@ -552,7 +552,9 @@ describe('libGD.js - GDJS Code Generation integration tests', function () {
     eventsFunction
       .getEvents()
       .unserializeFrom(project, eventsSerializerElement);
-    const group = eventsFunction.getObjectGroups().insert('MyObjectGroup', 0);
```

Comment:
Indeed, I think it worked because it just considered the first parameter to be an address, which was just read as being an ObjectGroup by the C++, and was inserted, but then any "garbage" read in memory was replaced anyway by the new name and objects). But it's mostly luck.